docs/bridge.md

```diff
@@ -240,8 +240,63 @@ In the above configuration replace
 
 > Note: By default the openidconnect app will use the email of the user to match the user from the oidc userinfo endpoint with the ownCloud account. So make sure your users have a unique primary email.
 
+### Proxy
+
+The Proxy is responsible for:
+- Routing authentication request to ocis-konnectd which is the default bundled OIDC-Provider
+- Routing frontend requests to ocis-phoenix
+- Routing backend requests to owncloud 10
+
+#### Get it!
+```console
+$ git clone git@github.com:owncloud/ocis-proxy.git
+$ cd ocis-proxy
+$ make
+```
+
+### Configuration
+
+Edit ./ocis-proxy/config/.proxy-example.json
+
+```json5
+{
+	"routes": [
+		{
+			"endpoint": "/",
+			"backend": "http://localhost:9100" //Should point to phoenix
+		},
+		{
+			"endpoint": "/.well-known/openid-configuration",
+			"backend": "http://localhost:9130" // Should point to ocis-konnectd
+		},
+		{
+			"endpoint": "/konnect/",
+			"backend": "http://localhost:9130" // Should point to ocis-konnectd
+		},
+		{
+			"endpoint": "/signin/",
+			"backend": "http://localhost:9130" // Should point to ocis-konnectd
+		},
+		{
+			"endpoint": "/ocs/v1.php/",
+			"backend": "http://localhost:4444" // Your owncloud instance
+		},
+		{
+			"endpoint": "/remote.php/webdav/",
+			"backend": "http://localhost:4444"  // Your owncloud instance
```

Comment:
@refs @butonic Not sure if we can add **https**://cloud.owncloud.com as a route here as this would result in a "nested" TLS-connection. If I remember correctly this will throw an error.

Reply:
🤔 indeed it will throw a error. I thought we agreed to instead deploy stuff behind the proxy, and not to in the other direction

Reply:
@refs I talked with @butonic we need to support this, as we should be able to talk to an existing oc installation with minimal changes to the existing configuration. As we have a custom director implementation now anyways it should be easy to transparently route https backends like here: https://github.com/micro/go-plugins/blob/master/micro/router/routes.go#L105-L120

Reply:
would that mean that we need to disable TLS on the proxy?

Reply:
@refs and I tested this against my live instance. we ran into other issues with apache https://github.com/owncloud/ocis-proxy/pull/7 but tls was not one of them ;-)

Reply:
No we should be able to terminate the backend response and then re-serve it via the proxy TLS